src/Orckestra.Composer.Search/Api/SearchController.cs

```diff
@@ -188,6 +192,19 @@ public virtual async Task<IHttpActionResult> SuggestCategories(AutoCompleteSearc
                 .Take(limit)
                 .ToList();
 
+
+            foreach (var category in finalSuggestions)
+            {
+                string url = CategoryBrowsingUrlProvider.BuildCategoryBrowsingUrl(new BuildCategoryBrowsingUrlParam
+                {
+                    CategoryId = category.Id,
+                    BaseUrl = RequestUtils.GetBaseUrl(Request).ToString(),
+                    CultureInfo = ComposerContext.CultureInfo,
+                    IsAllProductsPage = false
+                });
+                category.Url = url;
```

Comment:
we should not alwayst calculate Category Urls but just when we have parapeter true. See example how the limit parameter put to the request and try use it